projects/kubernetes-sigs/image-builder/Makefile

```diff
@@ -57,22 +57,32 @@ TARGETS_ALLOWED_WITH_NO_RELEASE_BRANCH=build release clean help binaries checksu
 TUFTOOL_TARGET=$(CARGO_HOME)/bin/tuftool
 BOTTLEROCKET_SETUP_TARGET=$(BOTTLEROCKET_DOWNLOAD_PATH)/bottlerocket-root-json-checksum
 
-FINAL_UBUNTU_OVA_PATH=$(ARTIFACTS_PATH)/ova/ubuntu.ova
-FINAL_UBUNTU_RAW_IMAGE_PATH=$(ARTIFACTS_PATH)/raw/ubuntu.gz
+FINAL_OVA_PATH=$(ARTIFACTS_PATH)/ova/$(IMAGE_OS).ova
+FINAL_RAW_IMAGE_PATH=$(ARTIFACTS_PATH)/raw/$(IMAGE_OS).gz
 FINAL_BOTTLEROCKET_OVA_PATH=$(ARTIFACTS_PATH)/ova/bottlerocket.ova
 FAKE_UBUNTU_OVA_PATH=$(IMAGE_BUILDER_DIR)/output/fake-ubuntu.ova
 FAKE_UBUNTU_RAW_PATH=$(IMAGE_BUILDER_DIR)/output/fake-ubuntu.gz
 
+IMAGE_OS?=ubuntu
 BUILD_AMI_TARGETS=build-ami-ubuntu-2004
-BUILD_OVA_TARGETS=setup-packer-configs-ova release-ova-bottlerocket $(FAKE_UBUNTU_OVA_PATH) $(FINAL_UBUNTU_OVA_PATH) $(FINAL_BOTTLEROCKET_OVA_PATH) upload-artifacts-ova
-BUILD_RAW_TARGETS=release-raw-ubuntu-2004-efi $(FAKE_UBUNTU_RAW_PATH) $(FINAL_UBUNTU_RAW_IMAGE_PATH) upload-artifacts-raw
+BUILD_OVA_TARGETS=setup-packer-configs-ova download-ova-bottlerocket $(FAKE_UBUNTU_OVA_PATH) $(FINAL_OVA_PATH) $(FINAL_BOTTLEROCKET_OVA_PATH) upload-artifacts-ova
+BUILD_RAW_TARGETS=release-raw-ubuntu-2004-efi $(FAKE_UBUNTU_RAW_PATH) $(FINAL_RAW_IMAGE_PATH) upload-artifacts-raw
 BUILD_TARGETS=$(BUILD_RAW_TARGETS) $(BUILD_AMI_TARGETS) $(BUILD_OVA_TARGETS)
 ifeq ($(IMAGE_FORMAT),ova)
-	S3_TARGET_PREREQUISITES=$(FINAL_UBUNTU_OVA_PATH) $(FINAL_BOTTLEROCKET_OVA_PATH)
-	RELEASE_TARGETS=release-ova-ubuntu-2004 release-ova-bottlerocket upload-artifacts-ova
+	ifeq ($(IMAGE_OS),ubuntu)
+		S3_TARGET_PREREQUISITES=$(FINAL_OVA_PATH) $(FINAL_BOTTLEROCKET_OVA_PATH)
+		RELEASE_TARGETS=release-ova-ubuntu-2004 download-ova-bottlerocket upload-artifacts-ova
```

Comment:
Can we add a comment here why we are piggy-backing if OS is ubuntu we want to run bottlerocket download/processing. Im for sure going to think why is this here few weeks later :P 

For later, maybe we can have a layer above IMAGE_OS, something like `VENDED_OS_RELEASE_TARGETS=ubuntu bottlerocket`. Or we can separate bottlerocket and have its own buildspec.

Reply:
Will do, I kinda wanted to put bottlerocket in a separate buildspec to make it cleaner. I will do a follow-up to this.